src/cheri-pte-ext.adoc

```diff
@@ -129,9 +135,9 @@ regards to the ongoing capability revocation cycle. Two schemes are permitted:
 
 * A load page fault exception is raised when a capability load or AMO instruction is executed
 and the virtual page's CRG bit does not equal <<sstatusreg_pte,sstatus>>.CRG.
-* A load page fault exception is raised when a capability load or AMO instruction is executed,
-the capability read from memory has tag set
-and the virtual page's CRG bit does not equal <<sstatusreg_pte,sstatus>>.CRG.
+* A load page fault exception is raised when a capability load or AMO instruction is executed
+and the virtual page's CRG bit does not equal <<sstatusreg_pte,sstatus>>.CRG
+and the capability read from memory has its tag set.
```

Comment:
Do we need to define here what happens if for loads via capabilities without <<c_perm>>? I imagine there is no need to trap if we aren't actually reading caps, but I do wonder if it could be a problem pulling tagged data into the cache. I don't think so since the next load with c_perm would perform this check again and then trap?

Reply:
I think we don't need to trap if there's no C-permission, and have clarified that in my latest commit.
@Timmmm thinks we do. @jonwoodruff / @jrtc27 can you add your opinion?

Reply:
Sorry I didn't mean that we do need to trap, I just meant your commit changed the behaviour whereas the rest of this PR is just clarifications, so I wasn't sure if it was intentional. Maybe take it out of this PR and make a new one?

Reply:
this isn't a change - it's the intention - I guess it's not what the SAIL does then?

Reply:
The spec already has this note although it doesn't mention Zcherilevels can also strip tags, so maybe this note should be adjusted
![image](https://github.com/user-attachments/assets/739f7288-534e-414e-98ca-17787dde5be1)

Reply:
Ah good spot. I think this change makes sense then. I am currently implementing Zcheripte in Sail.

Reply:
I'm trying to catch up here, and also found the note @jamie-melling mentioned.

As for adding further clarifications, if necessary, c_perm would be required to get a trap in both implementation options for CRG load checking (not just the second one).  That is, if a capability load or AMO instruction is executed, the authorizing capability has c_perm, and pte.CRG != csr.CRG.

On balance, I prefer the simple existing statement to adding more complexity to the conditions, but maybe people will find it confusing to have incomplete text!  Maybe pseudocode is the answer?

Reply:
Note updated